rails/client/webpack.config.js

```diff
@@ -149,6 +140,10 @@ const config = {
           },
         ],
       },
+      {
+        test: /\.md$/,
+        use: 'raw-loader',
```

Comment:
Does Kaiju need the raw loader?

Reply:
Yes, this is used to load our markdown files. This was just moved after the css assets. It was already part of this webpack configuration.